docs/nydusd.md

```diff
@@ -130,6 +130,9 @@ We are working on enabling cloud-hypervisor support for nydus.
   "iostats_files": true,
   // Enable support of fs extended attributes
   "enable_xattr": false,
+  // Amplified request batch size to read data from storage cache layer for user IO
```

Comment:
How about `Amplified user IO request batch size to read data from remote storage backend / local cache`

Reply:
Changed.